shared_sources/VMWCoreBackdoor.cpp

```diff
@@ -127,7 +127,7 @@ VMWCoreBackdoor::BackdoorRPCSend(regs_t* regs, char* data, size_t length)
 		"movl	$0x00010000, %%ebx	\n\t" // Init ebx
 
 		"cld						\n\t" // Backdoor call
-		"rep outsb (%%esi), (%%dx)	\n\t"
+		"rep outsb (%%rsi), (%%dx)	\n\t"
```

Comment:
Using `esi` previously was effectively zero-ing upper part of a 64-bit address, resulting in a page fault and a kernel panic.